Cargo.toml

```diff
@@ -7,3 +7,5 @@ description = "A client for the Minecraft RCON protocol."
 license = "GPL-3.0-or-later"
 
 [dependencies]
+clap = { version = "3.2", features = ["derive"] }
+regex = "1.6"
```

Comment:
Can we gate these two new dependencies behind a `cli` feature? https://doc.rust-lang.org/cargo/reference/features.html